<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="DESKTOP-14LIAQ3" ignored="0" name="Package with subpackages" tests="5" failures="3" timestamp="16 лист. 2017 22:17:47 GMT" time="16.632" errors="0">
  <testcase name="checkStatusCodeOfRequestWithInvalidCityName" time="3.307" classname="currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit">
    <failure type="java.lang.AssertionError" message="Actual statusCode doesn`t match with expected expected [404] but found [200]">
      <![CDATA[java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [404] but found [200]
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithInvalidCityName(WeatherDataForOneLocationByCityNameTestSuit.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCodeOfRequestWithInvalidCityName -->
  <testcase name="checkStatusCodeOfRequestWithMissedCityName" time="0.104" classname="currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit">
    <failure type="java.lang.AssertionError" message="Actual statusCode doesn`t match with expected expected [401] but found [200]">
      <![CDATA[java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithMissedCityName(WeatherDataForOneLocationByCityNameTestSuit.java:57)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCodeOfRequestWithMissedCityName -->
  <testcase name="checkStatusCodeOfRequestWithMissedCityNameAndAppId" time="0.08" classname="currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit">
    <failure type="java.lang.AssertionError" message="Actual statusCode doesn`t match with expected expected [401] but found [200]">
      <![CDATA[java.lang.AssertionError: Actual statusCode doesn`t match with expected expected [401] but found [200]
	at currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit.checkStatusCodeOfRequestWithMissedCityNameAndAppId(WeatherDataForOneLocationByCityNameTestSuit.java:65)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
... Removed 15 stack frames]]>
    </failure>
  </testcase> <!-- checkStatusCodeOfRequestWithMissedCityNameAndAppId -->
  <testcase name="checkStatusCodeOfRequestWithValidData" time="0.088" classname="currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit"/>
  <testcase name="checkWeatherIdWithValidData" time="0.39" classname="currentweatherdata.WeatherDataForOneLocationByCityNameTestSuit"/>
</testsuite> <!-- Package with subpackages -->
